tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Bot.Schema;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Core.Extensions.Tests
+{
+    public static class TranscriptUtilities
+    {
+        public static IEnumerable<IActivity> GetFromTestContext(TestContext context)
+        {
+            var transcriptsRootFolder = TestUtilities.GetKey("TranscriptsRootFolder") ?? @"..\..\..\..\..\transcripts";
+            var directory = Path.Combine(transcriptsRootFolder, context.FullyQualifiedTestClassName.Split('.').Last());
+            var fileName = $"{context.TestName}.transcript";
```

Comment:
I'm not clear on why we're loading transcript files, rather than .chat files and then on-the-fly converting them to transcript files. I'm worried this will create a maintenance and complexity problem

Reply:
We created the .chat files by hand and the used the [Chatdown](https://github.com/Microsoft/botbuilder-tools/tree/master/Chatdown) Node.js tool to generate .transcript files. Doing the same in a C# solution would involve either a dependency on Node.js and spawning a new process or rewriting the conversion tool in .NET.